lms/djangoapps/grades/tests/test_models.py

```diff
@@ -238,3 +239,58 @@ def test_update_or_create_grade(self, already_created):
         if already_created:
             self.assertEquals(created_grade.id, updated_grade.id)
             self.assertEquals(created_grade.earned_all, 6)
+
+
+@ddt.ddt
+class PersistentCourseGradesTest(GradesModelTestCase):
```

Comment:
You're missing a test case for the success path of the `read_course_grade` method.
Either create a new test method for it, or verify calls to it in the existing `test_update` and `test_create` methods.